nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/TailFile.java

```diff
@@ -251,6 +257,30 @@
             .defaultValue("false")
             .build();
 
+    static final PropertyDescriptor LINE_START_REGEX = new Builder()
+        .name("Line Start Regex")
+        .displayName("Line Start Regex")
```

Comment:
With the existing property named `Rolling Filename Pattern`, what do you think about naming this property `Line Start Pattern`?

Reply:
Yeah, good call.